packages/lab/src/tabs-next/TabNext.css

```diff
@@ -0,0 +1,120 @@
+/* Class applied to root Tab element */
+.saltTabNext {
+  --tabs-tab-background: var(--salt-navigable-primary-background);
+  --tabs-tab-background-hover: rgba(0, 0, 0);
+  --tabs-tab-backgroundOpacity-hover: 0.08;
+  --tabs-tab-cursor: pointer;
+  --tabs-tab-position: relative;
+}
+
+.salt-theme[data-mode="dark"] .saltTabNext {
+  --tabs-tab-background-hover: rgba(255, 255, 255);
+}
+
+.saltTabNext {
+  flex-shrink: 0;
+  align-items: center;
+  align-self: stretch;
+  justify-content: center;
+  background: var(--saltTabs-tab-background, var(--tabs-tab-background));
+  border: none;
+  border-radius: 0;
+  color: var(--salt-text-primary-foreground);
+  cursor: var(--saltTabs-tab-cursor, var(--tabs-tab-cursor));
```

Comment:
I would imagine `color` would be more customisable than `cursor` :)

Reply:
I added an override as part of the css api

Reply:
We can continue this convo on teams